lib/http.js

```diff
@@ -8,6 +8,7 @@
 var url = require('url');
 var req = require('request');
 var debug = require('debug')('node-soap');
+var httpntlm = require('httpntlm');
```

Comment:
Please, even if the required package does not respect `camelCase`, keep `camelCase`ing in the codebase and rename it to `httpNtlm`